oteapi_dlite/strategies/parse_image.py

```diff
@@ -0,0 +1,195 @@
+"""Strategy class for parsing xlsx to a DLite instance."""
+# pylint: disable=no-self-use,unused-argument
+from dataclasses import dataclass
+from pathlib import Path
+from random import getrandbits
+
+import numpy as np
+from PIL import Image
+from pydantic import BaseModel, Field, HttpUrl
+from typing import TYPE_CHECKING, Any, Dict, Optional, Tuple
+
+import dlite
+
+if TYPE_CHECKING:  # pragma: no cover
+    from typing import Any, Dict, Optional, Tuple
+
+    from oteapi.models.resourceconfig import ResourceConfig
+
+
+class DLiteImageConfig(BaseModel):
+    """Configuration for DLite image parser."""
+
+    def __init__(self, **kwargs) -> None:
```

Comment:
The __init__() function can be removed. It is not needed for pydantic data models.

Reply:
The point of this is to be able to include other keywords than just the three fields `crop, id, metadata`, by putting them into `DLiteImageConfig.configuration`. This isn't done automatically by pydantic.

I can add a test of `DLiteImageConfig` to demonstrate.

Reply:
This is done instead by setting the `extra` `Config` option ([docs](https://pydantic-docs.helpmanual.io/usage/model_config/#options)) or here you could subclass `AttrDict` as well.

Reply:
Yes, but that would make the extra attributes go into the class itself, not into its `configuration` field. For example, if one added an entry called `entry1` to the configuration of the `ResourceConfig` object that one initializes the `DLiteImageParseStrategy` object with, it would become `DLiteImageConfig.entry1`, rather than `DLiteImageConfig.configuration["entry1"]`.